zebra/zebra_fpm.c

```diff
@@ -2029,11 +2031,21 @@ static int zfpm_init(struct thread_master *master)
 	return 0;
 }
 
+static int zfpm_fini(void)
```

Comment:
should we do `stop_stats_timer` also, or is that one ok?

Reply:
fixed